src/games/strategy/engine/framework/startup/ui/MetaSetupPanel.java

```diff
@@ -124,55 +137,35 @@ private void layoutComponents() {
-      @Override
-      public void actionPerformed(final ActionEvent e) {
-        m_model.showServer(MetaSetupPanel.this);
-      }
-    });
-    m_connectToHostedGame.addActionListener(new ActionListener() {
-      @Override
-      public void actionPerformed(final ActionEvent e) {
-        m_model.showClient(MetaSetupPanel.this);
-      }
-    });
-    m_connectToLobby.addActionListener(new ActionListener() {
-      @Override
-      public void actionPerformed(final ActionEvent e) {
-        connectToLobby();
-      }
-    });
-    m_enginePreferences.addActionListener(new ActionListener() {
-      @Override
-      public void actionPerformed(final ActionEvent e) {
-        enginePreferences();
-      }
-    });
-    m_ruleBook.addActionListener(new ActionListener() {
-      @Override
-      public void actionPerformed(final ActionEvent e) {
-        ruleBook();
-      }
-    });
+    m_startLocal.addActionListener(e -> m_model.showLocal());
+    m_startTutorial.addActionListener(e -> launchTutorialMap(MetaSetupPanel.this));
+    m_startPBEM.addActionListener(e -> m_model.showPBEM());
+    m_hostGame.addActionListener(e -> m_model.showServer(MetaSetupPanel.this));
+    m_connectToHostedGame.addActionListener(e -> m_model.showClient(MetaSetupPanel.this));
+    m_connectToLobby.addActionListener(e -> connectToLobby());
+    m_enginePreferences.addActionListener(e -> enginePreferences());
+    m_ruleBook.addActionListener(e -> ruleBook());
     m_donate.addActionListener(e -> SwingComponents.newOpenUrlConfirmationDialog(UrlConstants.PAYPAL_DONATE));
-    m_about.addActionListener(new ActionListener() {
-      @Override
-      public void actionPerformed(final ActionEvent e) {
-        about();
-      }
-    });
+    m_about.addActionListener(e -> about());
+  }
+  
+  private static void launchTutorialMap(Component parent){
+    GameSelectorModel gameSelectorModel = new GameSelectorModel();
+    NewGameChooserEntry tutorialEntry = new NewGameChooserModel().findByName("Tutorial");
+    gameSelectorModel.load(tutorialEntry);
+    Map<String, Boolean> playersEnabled = tutorialEntry.getGameData().getPlayerList().getPlayersEnabledListing();
+    Map<String, String> playerTypes = new HashMap<>();
+    for(PlayerID player : tutorialEntry.getGameData().getPlayerList()){
+      playerTypes.put(player.getName(),
+          player.getName().startsWith("AI") || player.getName().startsWith("Neutral") ? "Hard (AI)" : "Human");
+    }
+    final PlayerListing playerListing =
+        new PlayerListing(null, playersEnabled, playerTypes, gameSelectorModel.getGameData().getGameVersion(),
+            gameSelectorModel.getGameName(), gameSelectorModel.getGameRound(), null, null);
+    ILauncher tutorialLauncher = new LocalLauncher(gameSelectorModel, new PlainRandomSource(), playerListing);
+    tutorialLauncher.launch(parent);
+    System.out.println("PUS" + (tutorialEntry.getGameData().getResourceList().getResource(Constants.PUS) != null));
```

Comment:
The tutorial game does launch. I would check the code path that launches the game normally, to see if there is something different there. My thought was that may PU was missing from the XML spec, but given the normal launch, probably not the case. You could add in some temporary "if (!null)" type of checksk to get past the issue temporarily.